packages/plugins/io-center/src/ui/components/object-tree/object-tree.svelte

```diff
@@ -1,14 +1,43 @@
-<object-tree>
+<script lang="ts">
+	import type { ObjectTree } from "./types.object-tree";
+	import { Accordion } from "@oscd-plugins/core-ui-svelte";
+	import type { TreeNode as TreeNodeType } from "./types.object-tree";
+	import TreeNode from "./tree-node.svelte";
 
-</object-tree>
+	type Props = {
+		objectTree: ObjectTree;
+	};
 
+	let { objectTree }: Props = $props();
 
-<script lang="ts">
-	import type { ObjectTree } from "./object-tree"
+	let selectedNodeName = $state("");
 
-	type Props = {
-		objectTree: ObjectTree
-	}
+	const getOpenItems = (nodes: TreeNodeType[]): string[] =>
```

Comment:
I am not big fan of arrow functions when they do not have any advantage.
They just confusing because we store them in a variable. Would you consider using functions? eg.:
`function getOpenItems(nodes: TreeNodeType[]){...}`